src/test/java/uk/gov/homeoffice/digital/sas/timecard/validators/timeentry/TimeEntryValidatorTest.java

```diff
@@ -50,6 +50,21 @@ void saveTimeEntry() {
                 getAsDate(EXISTING_SHIFT_END_TIME)));
     }
 
+    @Test
+    void validate_startTimeBeforeEndTime_errorReturned() {
+        var time = LocalDateTime.of(
+            2022, 1, 1, 6, 0, 0);
+        var startTime = getAsDate(time);
+        var endTime = getAsDate(time.minusMinutes(1));
+        var timeEntryNew = createTimeEntry(OWNER_ID_1, startTime, endTime);
+
+        Throwable thrown = catchThrowable(() -> saveEntryAndFlushDatabase(timeEntryNew));
+
+        assertThat(thrown).isInstanceOf(ConstraintViolationException.class);
+        assertPropertyErrorType((ConstraintViolationException) thrown, ClashingProperty.END_TIME);
+        assertThat(thrown.getMessage()).contains("End time must be after start time");
```

Comment:
Use constant in validator class, once added

Reply:
Constant added